src/Accelerators/NNPA/Runtime/zdnnx/omp_ops.c

```diff
@@ -73,13 +69,9 @@ uint32_t zdnnx_get_num_procs() {
   if (zdnnx_num_procs > 0)
     return zdnnx_num_procs;
 
-  zdnnx_num_procs = 0;
-  for (int i = 0; i < omp_get_num_places(); ++i)
-    zdnnx_num_procs += omp_get_place_num_procs(i);
-  if (zdnnx_num_procs == 0) {
-    // OMP_PLACES is not set. Return 1 to say using one processor.
-    zdnnx_num_procs = 1;
-  }
+  // If OMP_NUM_THREADS is not set, it is the sequential mode.
+  const char *env = getenv("OMP_NUM_THREADS");
+  zdnnx_num_procs = (env != NULL) ? atoi(env) : 1;
```

Comment:
What is the difference between `zdnnx_num_procs` and `zdnnx_num_zaius`. The names feel confusing and if they are two different entities, their name should be clearer.

Reply:
Good catch! `zdnnx_num_procs` is not necessary any more. I just removed it and revised some variable names. Thanks!